lib/services/insights/package.json

```diff
@@ -3,6 +3,7 @@
   "author": "Microsoft Corporation",
   "contributors": [
     "Kamath, Prarthana <pkamath@microsoft.com>",
+    "Medhekar, Shibani <shimedh@microsoft.com>",
```

Comment:
Can you update the package version over here? As you will publishing your individual package based on this package.json file.